packages/xo-web/src/xo-app/disk-import/index.js

```diff
@@ -0,0 +1,190 @@
+                name,
+                sr,
+                type,
+                vmdkData,
+              })
+            }
+          })
+        )
+        return { disks }
+      },
+      linkState,
+      onChangeDescription: (_, { target: { name, value } }) => ({
+        mapDescriptions,
+      }) => {
+        mapDescriptions[name] = value
+        return { mapDescriptions }
+      },
+      onChangeName: (_, { target: { name, value } }) => ({ mapNames }) => {
+        mapNames[name] = value
+        return { mapNames }
+      },
+      onChangeSr: (_, sr) => ({ sr }),
+      parseVmdk: async file =>
+        readCapacityAndGrainTable(async (start, end) => {
+          /* global FileReader */
+          const reader = new FileReader()
+          reader.readAsArrayBuffer(file.slice(start, end))
+          return (await fromEvent(reader, 'loadend')).target.result
+        }),
+      reset: getInitialState,
+    },
+  }),
+  injectIntl,
+  injectState,
+  class extends Component {
+    _import = () => {
+      const {
+        state: { disks, mapDescriptions, mapNames, sr },
+      } = this.props
+      return importDisks(
+        disks.map(disk => {
+          const { id, name } = disk
+          return {
+            ...disk,
+            name: mapNames[id] || name,
+            description: mapDescriptions[id],
+          }
+        }),
+        sr
+      )
+    }
+
+    render() {
+      const {
+        effects,
+        intl: { formatMessage },
+        state: { disks, mapDescriptions, mapNames, sr },
+      } = this.props
+      return (
+        <Container>
+          <form id='import-form'>
+            <Row>
+              <LabelCol>{_('importToSr')}</LabelCol>
+              <InputCol>
+                <SelectSr onChange={effects.onChangeSr} required value={sr} />
+              </InputCol>
+            </Row>
+            {sr !== undefined && (
+              <div>
+                <Dropzone
+                  onDrop={effects.handleDrop}
+                  message={_('dropDisksFiles')}
+                />
+                {disks.length > 0 && (
+                  <div>
+                    <div>
+                      {disks.map(({ file: { name, preview, size }, id }) => (
+                        <Collapse
+                          buttonText={`${name} - ${formatSize(size)}`}
+                          size='small'
+                          className='mb-1'
+                        >
+                          <div className='mt-1' key={preview}>
+                            <Row>
+                              <LabelCol>{_('formName')}</LabelCol>
+                              <InputCol>
+                                <input
+                                  className='form-control'
+                                  name={id}
+                                  onChange={effects.onChangeName}
+                                  placeholder={formatMessage(
+                                    messages.diskNamePlaceholder
```

Comment:
Why not using the filename as a placeholder?

Reply:
If I use a defined  `value` I can change it.

Reply:
What do you mean?